src/Microsoft.AspNet.Mvc.Core/ViewComponents/DefaultViewComponentActivator.cs

```diff
@@ -3,40 +3,38 @@
 
 using System;
 using System.Collections.Concurrent;
-using System.Collections.Generic;
 using System.Reflection;
-using Microsoft.AspNet.Mvc.Rendering;
-using Microsoft.Framework.DependencyInjection;
 using Microsoft.Framework.Internal;
 
 namespace Microsoft.AspNet.Mvc.ViewComponents
 {
     /// <summary>
-    /// Represents the <see cref="IViewComponentActivator"/> that is registered by default.
+    /// A default implemented of <see cref="IViewComponentActivator"/>.
```

Comment:
Suggest mentioning `[ViewComponentContext]` and fact activator handles only `ViewComponentContext` properties with that attribute.